docs/blog/newsletter-march-2020.rst

```diff
@@ -0,0 +1,140 @@
+.. post:: March 05, 2020
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – March 2020
+######################################
+
+Hello everyone! March is here (already?!), and so is a fresh new edition of the WTD newsletter.
+
+Firstly, announcing some news I know some of you have been waiting for. We're delighted that the `results of last year's salary survey </surveys/salary-survey/2019/>`__ are now live! Openness is a core value of this community, so it's great to be able to present the anonymized data for your perusing.
+
+On the conference side, it's exciting to see the `awesome roster of speakers for Portland </conf/portland/2020/news/announcing-speakers/#full-speaker-line-up>`__ - some seriously great talks to look forward to.
+
+We do also want to address the situation with COVID-19. Currently, *we have no plans to cancel* the Portland conference, but are planning to take additional precautions during the on-site events. We'll be watching the situation closely over the next few weeks, and taking advice from the Oregon health authorities and United States CDC as it comes. We will assess the situation again next week and provide updates as the situation changes. 
+
+Moving swiftly on - it's been a month of great conversations on Slack, so let's head on to our stories.
```

Comment:
I am really struggling with this segue from "serious pandemic" to "some nice stories!". Is this too light-hearted?

Reply:
I don't think it's too lighthearted. One idea -- you could kind of move the segue to the beginning of the COVID-19 paragraph. Something like...

Before we get to this month's stories, we do also want to address...

Then maybe you could just go straight from "...as the situation changes." to the first story. I guess that would be just as abrupt, but it might help avoid any interpretation of lightheartedness in terms of how seriously WTD is taking the COVID-19 situation.

Reply:
I tried that but then had a similar problem with how I pitched the opener. If the current situation feels okay to you, then I think I'll stick with this - I just wanted to make sure I got a second opinion.

Reply:
You could say something like 

> Getting back to more usual documentation related topics

or similar